okhttp/src/jvmTest/java/okhttp3/internal/http2/HttpOverHttp2Test.kt

```diff
@@ -2045,27 +2041,77 @@ class HttpOverHttp2Test {
       .proxy(server.toProxyAddress())
       .build()
 
-    val call = client.newCall(
-      Request(
-        server.url("/").resolve("//android.com/foo")!!
-      )
-    )
+    val url = server.url("/").resolve("//android.com/foo")!!
```

Comment:
Existing behaviour, but where does this format get handled?

Reply:
It’s in HttpUrl. Two leading slashes keeps the scheme only!